### PR TITLE
Added Dockerfile and Metaploy Config

### LINKED ISSUE #1: Containerization of the project
- Create a production Dockerfile to run the project.
- Create an NGINX configuration file so that the project can be run using [Metaploy](https://github.com/metakgp/metaploy).

### PR DESCRIPTION
**Pull request by harshkhandeparkar**
Closes #1

### CODE REVIEW

File: metaploy/postinstall.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+trap 'cleanup' SIGQUIT SIGTERM SIGHUP
+
+"${@}" &
+
+cp /heimdall.metaploy.conf /etc/nginx/sites-enabled
+
+wait $!
+
+echo "lmao"
```
Comment (chirag-ghosh) [inline metaploy/postinstall.sh:16]: much echo

**Review by chirag-ghosh — APPROVED** [review]
LGTM

File: metaploy/postinstall.sh (hunk 1 above)
Comment (harshkhandeparkar) [reply metaploy/postinstall.sh:16]: lmaoo forgot to remove it. was added for debugging.

Comment (chirag-ghosh) [reply metaploy/postinstall.sh:16]: i thought it was a joke. doesnt matter anyway

Comment (harshkhandeparkar) [reply metaploy/postinstall.sh:16]: Today's mistake, tomorrow's easter egg.

**Pull request merged**